### ISSUE TITLE
Hercules r command incorrect output

### ISSUE THREAD

**Issue by ghost** [rbowler/spinhawk#35]
Just give r 8ffe.20 as I did

15:31:56 r 8ffe.20
15:31:56 R:0000000000008FFE:K:06=0000                                 ..
15:31:56 R:000000000000900E:K:06=0F10 11121314 15161718 191A1B1C 1D1E ................

It seems that going over to the new page the offset is forgotten to clear!

**Comment by rbowler** [comment]
Yes I see what you mean. Each line of the display cannot cross a page boundary. I cannot remember the reason for this but it was probably in case the storage key was different on the next page. Logically you would expect then that the next line would continue from 9000 for a length 1E bytes, but as you point out, Hercules seems to have forgotten to clear the offset.
